jobs/build/refresh-images/Jenkinsfile

```
@@ -315,14 +316,25 @@ Jenkins job: ${env.BUILD_URL}
     }
 
     stage ('Attach Images') {
+        def command_str = ""
         if (ADVISORY_ID != "") {
-            try {
-                buildlib.elliott """
+            if (ADVISORY_ID == "default") {
```

I would DRY this up to:
```suggestion
            def attach = ADVISORY_ID == "default" ? "--use-default-advisory image" : "--attach ${ADVISORY_ID}"
            try {
                buildlib.elliott """
 --group 'openshift-${OSE_MAJOR}.${OSE_MINOR}'
 find-builds
 --kind image
${attach}
"""
            }
```